static/locales/sr.yaml

```diff
@@ -165,7 +166,14 @@ Settings:
       Amber: 'Амбер'
       Orange: 'Наранђасто'
       Deep Orange: 'Тамно нарађасто'
-    Secondary Color Theme: 'Секундарна боја теме'
+      Dracula Cyan: 'Дракула Цијан'
+      Dracula Green: 'Дракула Зелено'
+      Dracula Orange: 'Дракула Наранђасто'
+      Dracula Pink: 'Дракула Розе'
+      Dracula Purple: 'Дракула Љубичасто'
+      Dracula Red: 'Дракула Црвено'
+      Dracula Yellow: 'Дракула Жуто'
+    Secondary Color Theme: 'Дракула боја теме'
```

Comment:
I'm gonna guess you didn't mean to do this change.

Reply:
Good catch. Thankfully, It seems to have just reverted to an older translation for that string, and shouldn't pose that significant of a problem.